ui-library-filesharing-ui-components/app/src/App.tsx

```diff
@@ -0,0 +1,65 @@
+import { ChatThreadClient } from "@azure/communication-chat";
+import { AzureCommunicationTokenCredential } from "@azure/communication-common";
+import {
+  createStatefulChatClient,
+  FluentThemeProvider,
+  ChatClientProvider,
+  ChatThreadClientProvider,
+  DEFAULT_COMPONENT_ICONS,
+} from "@azure/communication-react";
+import { registerIcons, initializeIcons } from "@fluentui/react";
+import React from "react";
+import ChatComponents from "./ChatComponents";
+import { initializeFileTypeIcons } from "@fluentui/react-file-type-icons";
+
+function App(): JSX.Element {
+  initializeIcons();
+  registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
+  initializeFileTypeIcons();
+
+  const endpointUrl = "";
```

Comment:
for other samples is it just an empty string or do we have commented placeholders so people understand what they should be entering?